detect/git.go

```diff
@@ -1,192 +1,32 @@
 package detect
 
 import (
-	"bytes"
-	"errors"
-	"fmt"
-	"net/url"
-	"os/exec"
-	"regexp"
-	"strings"
-	"time"
-
-	"github.com/gitleaks/go-gitdiff/gitdiff"
+	"context"
 
 	"github.com/zricethezav/gitleaks/v8/cmd/scm"
-	"github.com/zricethezav/gitleaks/v8/logging"
 	"github.com/zricethezav/gitleaks/v8/report"
 	"github.com/zricethezav/gitleaks/v8/sources"
 )
 
+type RemoteInfo sources.RemoteInfo
```

Comment:
It's not immediately obvious why this is done. Worth a comment explaining it's there for backwards compatibility.